src/Spago/Build.hs

```diff
@@ -84,11 +84,11 @@ build BuildOptions{..} maybePostBuild = do
     NoInstall -> pure ()
   let allGlobs = Packages.getGlobs deps depsOnly configSourcePaths <> sourcePaths
       buildAction = do
-        case alternateBackend of 
+        case alternateBackend of
```

Comment:
Hope you don't mind these whitespace changes, they're done by stylish-haskell. But I can undo them if you prefer.

Reply:
No worries, whitespace changes are fine!